ledger-api/grpc-definitions/com/daml/ledger/api/v1/update_service.proto

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v1;
+
+import "com/daml/ledger/api/v1/ledger_offset.proto";
+import "com/daml/ledger/api/v1/transaction.proto";
+import "com/daml/ledger/api/v1/transaction_filter.proto";
+import "com/daml/ledger/api/v1/transfer.proto";
+
+option java_outer_classname = "UpdateServiceOuterClass";
+option java_package = "com.daml.ledger.api.v1";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1";
+
+// Allows clients to read updates (transactions and transfers) from the ledger.
+service UpdateService {
+
+  // Read the ledger's filtered transaction stream and related transfers for a set of parties.
+  // For transactions it lists only creates and archives, but not other events.
+  // Omits all events on transient contracts, i.e., contracts that were both created and archived in the same transaction.
+  rpc GetUpdates (GetUpdatesRequest) returns (stream GetUpdatesResponse);
```

Comment:
Do we want the streaming endpoint for the ledger end here ?

Reply:
That would be something like a new concept of the endpoint which I would rather avoid at this time if not absolutely necessary.
In research we do not have such endpoint as well, no?
Do you think it is necessary to introduce something like that at this point?

Reply:
I am currently introducing something in research following the plan written here: https://github.com/DACH-NY/canton/issues/12614
Now, we may decide to have it only in research and not port it to LAPI if you have something that is more performant than my implementation (I am sure you do! :D )

Reply:
Left a comment there, this might be something we would need to discuss first.
For this particular issue we could introduce a stream if it makes sense, but that would impose design/implementation/integration effort on both ends. Not sure we want to tackle this now.
Let's discuss this

Reply:
As discussed with @cocreature this is no needed with the advent of the merged domain streams